.changeset/hot-pants-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-fetch": minor
```

Comment:
This is a patch because it doesn’t introduce a breaking change (that I’m aware of)

```suggestion
"openapi-fetch": patch
```

Reply:
According to semver:
```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```

Reply:
That only applies for > 1.0! For 0.x, it’s more standard to smush bugfixes/features into patches, and use minor for breaking changes. It helps people plan more accordingly. Semver dictates that 0.x releases are allowed to make breaking changes at any time, and even though this method isn’t “official,” it’s a very common pattern, and one that openapi-fetch has followed since the beginning.

But you are correct—that becomes the pattern as soon as 1.0 is released and thereafter

Reply:
I was not aware of this! Thanks for letting me know, I'll update that